packages/amplify_datastore/lib/amplify_datastore.dart

```diff
@@ -375,4 +375,14 @@ class NativeAmplifyApi
       _subscriptionsCache.remove(subscriptionId);
     }
   }
+
+
+  // Amplify.DataStore.Stop() callback
+  // Clean up subscriptions on stop.
```

Comment:
nit: this can be a doc comment
```suggestion
  /// Amplify.DataStore.Stop() callback.
  ///
  /// Cleans up subscriptions on stop.
```